notebooks/article_1.py

```diff
@@ -0,0 +1,1083 @@
+#!/usr/bin/env python
```

Comment:
I still find a lot of mistakes.
Can you please pass grammarly on the whole article?